.github/workflows/similarIssues.yml

```diff
@@ -0,0 +1,32 @@
+name: GitGudSimilarIssues comments
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  getsimilarissues:
+    runs-on: ubuntu-latest
+    outputs:
+      message: ${{ steps.getbody.outputs.message }}
+    steps:
+      - uses: actions/checkout@v2

```

Comment:
Does this script really need to checkout something?

Reply:
@craigloewen-msft cause I have no idea

Reply:
I believe it does to get the `app.js` file that runs for this action. Is it a problem to do the check out? If so I mean you can try and remove it and see what happens?

Reply:
`app.js` isn't part of our repo though. We shouldn't need to check out _our_ repo :gri:

Reply:
Ah user error when creating this repo. That line isn't needed! :)
```suggestion
```